webapp/src/js/enketo/widgets/mrdt.js

```diff
@@ -47,25 +47,34 @@ define( function( require, exports, module ) {
 
         if ( !service.enabled() ) {
             $translate( 'mrdt.disabled' ).then(function( label ) {
-                $el.append( '<p>' + label + '</p>' );
+                $el.find( '.or-appearance-mrdt-image' )
+                   .append( '<p>' + label + '</p>' );
             });
             return;
         }
 
         $el.on( 'click', '.btn.mrdt-verify', function() {
-            service.verify().then( function(image) {
+            service.verify().then( function(data) {
+                const items = data.split('|');
```

Comment:
I thought in the product call this week we said these mrdt changes were backward compatible? This doesn't look backward compatible ... the new data arriving from `service.verify()` could not be used as an img `src` element as-is currently being done in master. 

We will need to guarantee the `webapp` update before the `rdt app` update. Can you confirm this is the plan and this has been clearly communicated to Muso?

Reply:
Is there a way to make this backward compatible? I see the code in `medic-android` is pushing to `get('AndroidApi').v1` ... could there be a `v2`?

Reply:
I have implemented a simple change that guarantees that 3.6 works with the current version of the wrapper but I will also ask Muso about their upgrade plans